sound/soc/sof/pm.c

```diff
@@ -359,11 +359,16 @@ static int sof_suspend(struct device *dev, bool runtime_suspend)
 #endif
 	/* notify DSP of upcoming power down */
 	ret = sof_send_pm_ipc(sdev, SOF_IPC_PM_CTX_SAVE);
-	if (ret < 0) {
+	if (ret == -EBUSY || ret == -EAGAIN) {
 		dev_err(sdev->dev,
-			"error: ctx_save ipc error during suspend %d\n",
+			"ctx_save ipc error during suspend %d\n",
```

Comment:
I don't think you're getting a lllikkke for this one - we very much want to stick to the "error: ..." error message format :-) I'll drop this change

Reply:
@lyakh maybe :) other parts of kernel don't seem to add "error: " in the message... you are already classifying it as error with dev_err() so putting another one is not adding much. if you grep linux/sound/ for dev_err() usage, only soc/sof/ is using it like this...

Reply:
@kv2019i Yes, I don't like it either. I'd be happy if you submit a patch to revert them all and if it gets accepted! It's a waste of screen real estate...